TTHAnalysis/python/tools/nanoAOD/susySOS_modules.py

```diff
@@ -301,6 +343,8 @@ def tightLepWZ(lep,year):
 isTightLepVV = lambda : ObjTagger('isTightLepVV', "LepGood", [ lambda lep,year: tightLepVV(lep,year) ]) # relax iso cut
 isTightLepWZ = lambda : ObjTagger('isTightLepWZ', "LepGood", [ lambda lep,year: tightLepWZ(lep,year) ]) # relax iso cut
 
+isFOLep = lambda : ObjTagger('isFOLep', "LepGood", [ lambda lep,year: new_tightLeptonSel_SOS(lep,year) or  new_clean_and_FO_selection_SOS(lep,year) ]) 
+
 eleSel_seq = [isVLFOEle, isTightEle]
 tightLepCR_seq = [isTightLepDY,isTightLepTT,isTightLepVV,isTightLepWZ]
```

Comment:
Can we make a sequence for all of the modules that need to be run for the friend trees, instead of calling individual modules when running the command (e.g. see [here](https://twiki.cern.ch/twiki/bin/viewauth/CMS/SusySoftOS#MC))? The way it is done now is very error-prone and makes us forget modules all the time.